packages/vscode/src/utils.ts

```diff
@@ -97,3 +100,9 @@ export function isSubdir(parent: string, child: string) {
   const relative = path.relative(parent, child)
   return relative && !relative.startsWith('..') && !path.isAbsolute(relative)
 }
+
+async function isColorUtility(uno: UnoGenerator, utilName: string) {
+  const css = (await getPrettiedCSS(uno, utilName)).css
```

Comment:
Do we need to prettify this? We could probably use `uno.genreate` directly.

Reply:
I think result.css is a long string contains all css generated.
https://github.com/unocss/unocss/blob/0ba0d5a10e9ccd6c1da98dbea99cf9ddd94458b6/packages/vscode/src/annotation.ts#L98

Considering different pattern of css class name, It's hard to drag css of one utility. 

css define could be:
```
1. [font~="black"]{font-weight:900;}  (<div font="black" text-red >)

2. .font-black,[font-black=""]{font-weight:900;}   (<div font-black text-red >)
```

We could use `font-black` and Regex  to get utility css from `result.css` in pattern 2
But need more operation for pattern 1. 
So I am using prettify function just for get the utility css only.
Not for the prettify :)

Reply:
I see. We could directly call `uno.generate` again with that utility, it's been cached by UnoCSS and it should be fast. We don't need prettify on this.